share/models/base.py

```diff
@@ -166,11 +144,11 @@ class Meta:
 class ShareObject(models.Model, metaclass=ShareObjectMeta):
     id = models.AutoField(primary_key=True)
     objects = ShareObjectManager()
-    versions = VersionManager()
     changes = fields.GenericRelationNoCascade('Change', related_query_name='share_objects', content_type_field='target_type', object_id_field='target_id', for_concrete_model=True)
 
     class Meta:
         abstract = True
+        base_manager_name = 'objects'
```

Comment:
`base_manager_name` is the [recommended](https://docs.djangoproject.com/en/1.11/topics/db/managers/#using-managers-for-related-object-access) way to have a custom `_base_manager`. Setting `_base_manager` directly might work too, but that seems like a rug that could be pulled out from under us in a future upgrade.

And yes, that's why the Metas inherit.